pilot/pkg/serviceregistry/kube/controller.go

```diff
@@ -199,6 +232,43 @@ func (c *Controller) createInformer(
 	return cacheHandler{informer: informer, handler: handler}
 }
 
+// createEDSInformer is a special informer for Endpoints
+func (c *Controller) createEDSInformer(
```

Comment:
I found no differences by diffing this against `createInformer`. Why bother to create a new one if they're the same?

Reply:
This informer doesn't add to the queue - just calls updateEDS (which has its lock for multi thread).

Reply:
In general - trying to move away from the queue, it behaves pretty badly in many ways and over complex.

Reply:
Yes *does* push stuff to the queue, see my comments below.

Reply:
nvm. I was looking at a outdated snapshot.

If the difference is only about queue or not, does it worth it copying paste and give the handler a new name? Rather than just passing an argument to the method.